src/app/app-routing.module.ts

```diff
@@ -7,10 +7,7 @@ const routes: Routes = [
       redirectTo: 'login',
       pathMatch: 'full'
   },
-  {
-    path:'',
```

Comment:
There is already a default path set above, so I removed this one to avoid conflict.